packages/lesswrong/components/comments/CommentPermalink.tsx

```diff
@@ -67,6 +67,7 @@ const CommentPermalink = ({ documentId, post, classes }: {
       showPostTitle: false,
     },
     expandByDefault: true,
+    noAutoScroll: true
```

Comment:
This broke within-page comment permalinks on post pages (it makes it so the comment appears at the top of the page but it doesn't scroll up)